rero_mef/api_mef.py

```diff
@@ -115,47 +115,47 @@ def get_pids_with_multiple_mef(cls, record_types=[], verbose=False):
         pids = {}
         multiple_pids = {}
         missing_pids = {}
-        for record_type in record_types:
-            if verbose:
-                click.echo(f'Calculating {record_type}:')
-            pids[record_type] = {}
-            multiple_pids[record_type] = {}
-            missing_pids[record_type] = []
-
-            if agent_class := get_entity_class(record_type):
-                agent_name = agent_class.name
-                query = cls.search().filter('exists', field=agent_name)
-                progress = progressbar(
-                    items=query
-                    .params(preserve_order=True)
-                    .sort({'pid': {'order': 'asc'}})
-                    .scan(),
-                    length=query.count(),
-                    verbose=verbose
-                )
-                for hit in progress:
-                    data = hit.to_dict()
-                    mef_pid = data['pid']
-                    agent_pid = data[agent_name]['pid']
-                    pids[record_type].setdefault(agent_pid, [])
-                    pids[record_type][agent_pid].append(mef_pid)
-                    if len(pids[record_type][agent_pid]) > 1:
-                        multiple_pids[record_type][agent_pid] = \
-                            pids[record_type][agent_pid]
-                if len(pids[record_type]) < agent_class.count():
+        if record_types:
```

Comment:
I do not think you need this `if` statement.

Reply:
We need this because record_types could be None.